### PR TITLE
Machine type selector macOS version indicator update [PLANG-1641] [PLANG-1645]

### PR DESCRIPTION
**Pull request by erosdome**

### CODE REVIEW

File: rails/strings.rb [hunk 1]
```diff
@@ -692,7 +692,8 @@ def strings
                 machine_type_notes: {
                     gen1: "The tried and tested machines that have been serving our customers for many years now",
                     gen2: "These machines run on a new hardware generation, offering improved performance compared to Gen1 hardware"
-                }
+                },
+                m1_elite_xl_beta_warning: "M1 Elite XL machines are in beta. Some features, like the Android emulator, don't work on it yet."
```
Comment (gaborszakacs) [inline rails/strings.rb:696]: Is this the official, signed-off copy? I'd say "supported", not "work", but it's not a blocker :) 

```suggestion
                m1_elite_xl_beta_warning: "M1 Elite XL machines are in beta. Some features (e.g. the Android emulator) are not supported yet."
```

File: source/stylesheets/_tab-stack.scss.erb [hunk 2]
```diff
@@ -209,7 +206,7 @@
 									svg {
 										width: 16px;
 										height: 18px;
-										fill: $def-darkgrey;
+										fill: #201b22;
```
Comment (gaborszakacs) [inline source/stylesheets/_tab-stack.scss.erb:209]: Do we usually hardcode stuff like this? 😇

File: source/stylesheets/_tab-stack.scss.erb [hunk 3]
```diff
@@ -259,66 +256,29 @@
...
+							@include margin-y(8px);
 							background-color: $def-verylightgrey;
 						}
 
 						.bottom {
 							display: flex;
-							align-items: center;
-							width: 100%;
-
-							.cpu {
```
Comment (gaborszakacs) [inline source/stylesheets/_tab-stack.scss.erb:289]: I saw a lot of deletions from CSS, the new layout doesn't need them at all?

Comment (erosdome) [reply source/stylesheets/_tab-stack.scss.erb:289]: Nope, they were simple inline texts not needing any special styling.

File: source/templates/stack.slim [hunk 4]
```diff
@@ -32,23 +32,27 @@
...
+									.specs
+										.cpu
+											span[ng-bind="machineType.cpuCount"]
+											span<> @
+											span[ng-bind="machineType.cpuDescription"]
+										.ram[ng-bind="machineType.ram"]
+									.os[ng-if="appService.stack.osForChip" ng-bind="stackCtrl.osForStackWithMachineType(appService.stack, machineType)"]
 					/ a.machine-type-info-link[ng-href="{{ cpuDescriptionURL }}" target="_blank"] == data[:strings][:machine_type][:learn_more_about]
+				.machine-type-warning-wrapper
+					p.notification.warning[ng-if="appService.defaultMachineType.id === 'g2-m1.8core'"] == data[:strings][:stack][:m1_elite_xl_beta_warning]
```
[the remark this answers is not in the document]
Comment (erosdome) [reply source/templates/stack.slim:55]: Nope, only for this one it seems.

**Review by gaborszakacs — APPROVED** [review]

**Pull request merged**